### PR TITLE
more word change

### PR DESCRIPTION
**Pull request by fab144**

### CODE REVIEW

File: android/assets/jsons/translations/Malay.properties [hunk 1]
```diff
@@ -4861,17 +4862,17 @@ Singapore = Singapura
...
 We never fully trusted you from the start. = 
  # Requires translation!
-Tyre = 
+Tyre = Tyre
 
  # Requires translation!
 May the Heavens forgive you for inflicting this humiliation to our people. = 
  # Requires translation!
-Zanzibar = 
+Zanzibar =Zanzibar
```
Comment (will-ca) [inline android/assets/jsons/translations/Malay.properties:4870]: Need a space after the equals sign.

Hm. We should have a unit test for that.

Comment (will-ca) [reply android/assets/jsons/translations/Malay.properties:4870]: Meh. I guess it's a no-op anyway.

**Comment by SomeTroglodyte** [comment]
Identity translations are still important to get rid of the 'requires' generation and to show the translator has - hopefully - thought about it and made a conscious decision.

**Comment by yairm210** [comment]
merge translations

**Comment by yairm210** [comment]
Translations branch created

**Comment by yairm210** [comment]
Translations PR created

**Pull request merged**